trustmanager/filestore_test.go

```diff
@@ -50,7 +50,99 @@ func TestAddFile(t *testing.T) {
 	}
 }
 
-func TestRemoveFile(t *testing.T) {
+func TestAddAndGetEncrypted(t *testing.T) {
```

Comment:
wrong key is different. since we're using a counter mode bit flips allow for changing the ciphertext and getting a predictable change in the decrypted plaintext -- that is, bit flip in ciphertext leads to exact same bit flipped in the decrypted plaintext